src/diva/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/diva.js/6.0.2/js/diva.js"></script>
```

Comment:
If we change the folder `diva.js` to `divajs` this will need updated too. Also, I'm a little concerned about the `6.0.2` being hardcoded here, as this will break when diva is updated. The best solution would probably be to specify a tagged version.

`git clone --branch v6.0.2 git://github.com/DDMAL/diva.js.git divajs`

If we're cloning from master on the other projects and they build to a versioned directory, we should make a similar change there as well.